src/index.js

```diff
@@ -34,18 +35,23 @@ if ( !! window.wcCalypsoBridge.isEcommercePlan ) {
 		),
 	} );
 
+	registerPlugin( 'wc-calypso-bridge-homescreen-slotfill-banner', {
```

Comment:
`window.wcCalypsoBridge.isEcommercePlan`  only detects ECOM plan. 

I think once this [PR](https://github.com/Automattic/wc-calypso-bridge/pull/926/files#diff-c9506099f3f037eaa3fc4b48305a4d641f4204266a5494640c4876e942ea711a) gets approved, we should add `window.wcCalypsoBridge.isFreeTrialPlan` and use that instead.

Reply:
Thanks! Good catch 😅

Reply:
@rjchow The [PR](https://github.com/Automattic/wc-calypso-bridge/pull/926/files#diff-c9506099f3f037eaa3fc4b48305a4d641f4204266a5494640c4876e942ea711a) is now merged